src/sentry/api/endpoints/project_rule_details.py

```diff
@@ -11,19 +11,12 @@
 from sentry.api.serializers.rest_framework.rule import RuleSerializer as DrfRuleSerializer
 from sentry.integrations.slack.utils import RedisRuleStatus
 from sentry.mediators import project_rules
-from sentry.models import (
-    RuleActivity,
-    RuleActivityType,
-    RuleSnooze,
-    RuleStatus,
-    SentryAppComponent,
-    Team,
-    User,
-)
+from sentry.models import RuleActivity, RuleActivityType, RuleStatus, SentryAppComponent, Team, User
 from sentry.models.integrations.sentry_app_installation import (
     SentryAppInstallation,
     prepare_ui_component,
 )
+from sentry.models.rulesnooze import RuleSnooze
```

Comment:
i'm curious why the rest of the models don't have this import issue https://github.com/getsentry/sentry/blob/5d5f162ad8bd85de5f3ebabcd3d2755381152532/src/sentry/models/__init__.py#L101 -- because its not `import *`?

Reply:
yep that's indeed why.  an explicit re-export can be done with `from a import b as b` -- but there's a long-term goal to make imports explicit anyway (to eliminate the import-cycle hell we've got right now)